pkgs/development/python-modules/stack-data/default.nix

```
@@ -1,55 +1,43 @@
-{ asttokens
-, buildPythonPackage
-, cython
-, executing
+{ buildPythonPackage
 , fetchFromGitHub
-, git
 , lib
-, littleutils
-, pure-eval
-, pygments
+, python3
+, pythonPackages
 , pytestCheckHook
-, setuptools-scm
-, toml
-, typeguard
 }:
 
 buildPythonPackage rec {
   pname = "stack-data";
-  version = "0.2.0";
-  format = "setuptools";
+  version = "0.6.3";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "alexmojaki";
     repo = "stack_data";
     rev = "v${version}";
-    hash = "sha256-brXFrk1UU5hxCVeRvGK7wzRA0Hoj9fgqoxTIwInPrEc=";
+    hash = "sha256-dmBhfCg60KX3gWp3k1CGRxW14z3BLlair0PjLW9HFYo=";
   };
 
-  nativeBuildInputs = [
-    git
-    setuptools-scm
-    toml
-  ];
-
-  propagatedBuildInputs = [
+  propagatedBuildInputs = with pythonPackages; [
     asttokens
     executing
     pure-eval
   ];
 
-  nativeCheckInputs = [
-    cython
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.setuptools-scm
+    python3.pkgs.wheel
+  ];
```

```suggestion
  build-system = [
    setuptools
    setuptools-scm
    wheel
  ];
```

Some people suggest to remove `setuptools` as it will be pulled by `setuptools-scm`. Personally I'm fine with listing all build requirements.